src/Corax/Queries/MultiTermMatch.cs

```diff
@@ -104,7 +104,7 @@ public int Fill(Span<long> buffer)
 
         [SkipLocalsInit]
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
-        public int AndWith(Span<long> buffer)
+        public int AndWith(Span<long> buffer, int matches)
```

Comment:
Just to make sure - don't we need to use `matches` somewhere inside that method?

Reply:
Good catch, not a difference in term of correctness but I was doing more work that I was intending to do. Fixed now